streams/sidecar_file.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "basic_types.proto";
+import "timestamp.proto";
+import "contract_call_local.proto";
+
+/**
+ * A SidecarFile contains a list of TransactionSidecarRecords that are all created
+ * in the same period and related to the same RecordStreamFile.
+ */
+message SidecarFile {
+
+  /**
+   * List of sidecar records
+   */
+  repeated TransactionSidecarRecord sidecar_records = 1;
+}
+
+/**
+ * TransactionSidecarRecord is used to create sidecar records complementing
+ * TransactionRecord and storing additional information about a transaction's execution.
+ */
+message TransactionSidecarRecord {
+  /**
+   * Consensus timestamp will be the same as the consensus timestamp of the
+   * transaction the side car is related to. This offers a convenient
+   * way to match record to sidecar.
+   */
+  Timestamp consensus_timestamp = 1;
+
+  /*
+  * List of sidecar types.
+  * In future there will be other categories.
+  */
+  oneof sidecar_records {
+    ContractStateChange state_changes = 2;
```

Comment:
This should be a list of contract state changes. We may need to define it in a separate message like:
message ContractStateChanges {
  repeated ContractStateChange contract_state_changes = 1;
}